wait/host_port_test.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/testcontainers/testcontainers-go/exec"
+	"github.com/testcontainers/testcontainers-go/log"
```

Comment:
question: I usually alias the testcontainers packages with the `tc` prefix instead: `tclog "github.com/.../log"`, so the stdlib `log` would remain the same.

BUT I'm not sure if this idiomatic and should be done exactly the other way around (as you suggested: library packages go first).

What is the idiomatic way of doing this?

Reply:
I only do it if there is a conflict.

Reply:
Indeed, just in case of conflicts, as in this file. Else, keep it without the alias

Reply:
fix pushed :)